db/schema.rb

```diff
@@ -22,16 +22,17 @@
     t.index ["user_id"], name: "index_authentications_on_user_id"
   end
 
-  create_table "reservations", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4", force: :cascade do |t|
+  create_table "reservations", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci", force: :cascade do |t|
     t.bigint "user_id", null: false
     t.integer "event_date_id", null: false
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.integer "event_slot_id", default: 0, null: false
```

Comment:
Please don't give this a default of 0. That's like a more confusing way of letting it be null. Just let the column be null.